src/auth.ts

```diff
@@ -10,8 +10,8 @@ export async function getSignUpUrl(returnPathname?: string) {
   return getAuthorizationUrl({ returnPathname, screenHint: 'sign-up' });
 }
 
-export async function signOut(request: Request) {
-  return await terminateSession(request);
+export async function signOut(request: Request, returnTo?: string) {
```

Comment:
Worth doing an options object here with a `returnTo`, [similar to the Next SDK](https://github.com/workos/authkit-nextjs/blob/657e317a83a7b02d91e2eaea988b952094d92de1/src/auth.ts#L32)?

I don't have a strong opinion, but might be good to maintain consistency, or in case later we add other options.

Reply:
That's a good call. Would be less intrusive later if there is another option to pass.